R/google_trends.R

```diff
@@ -76,17 +76,31 @@ getGoogleTrends <- function(user, password, query = "", type = "trend", last = "
     # So the first result is enough
     key <- keys[startsWith(keys, "Top.regions") | startsWith(keys, "Top.subregions")][[1]]
     # gather columns except for the first column (names of region) to make it easy to visualise
+
+    # if query is single, geo is used for a column name, so convert it to the query
```

Comment:
The comment is too terse to convey the idea of what is going on.
You could write something like,

If query consists of only one word, gtrendsR uses regions (e.g. Japan) for output column name instead of the query word.
So we are replacing it with the query word so that we can use the query word in subsequent steps.